src/subsystems/haskell/builders/simple-haskell/default.nix

```diff
@@ -125,6 +125,9 @@ in {
           doCheck = false;
           doBenchmark = false;
 
+          # FIXME: this seems to skip over the default package
+          configureFlags = subsystemAttrs.cabalFlags."${name}" or [];
```

Comment:
In the `cabal.project.freeze` files I generated, I generally noticed that only one version of a package ended up being prescribed, so I didn't bother to key the attrset according to both name and version. 

But I should probably reshape this anyway since I think in general Haskell closures can end up with more than one version of a package.

Is there a way of specifying the version of the `defaultPackage` that is cleaner than simply adding another entry to `extraArgs`?